modules/fs2/rules/src/main/scala/org/typelevel/fix/SyncCompiler.scala

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2022 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.typelevel.fix
+
+import scalafix.v1._
+
+import scala.meta._
+
+class SyncCompiler extends SemanticRule("TypelevelSyncCompiler") {
```

Comment:
I feel like`Fs2SyncCompiler` or similar is a better name, but that would be inconsistent with all our other rules. I still think the `Typelevel*` prefix makes sense, since its clear which project the rules originated from.

Reply:
`TypelevelFs2SyncCompiler`? `TlFs2SyncCompiler`?

Reply:
@bplommer thanks for chiming in! See also discussion in https://github.com/typelevel/typelevel-scalafix/pull/8.

Reply:
I think `TypelevelFs2SyncCompiler` is the right name. My inner perfectionist hates that its inconsistent with `TypelevelMapSequence` and `TypelevelUnusedIO` which don't mention Cats or Cats Effect, but maybe that's ok: map/sequence and `IO` are much less ambiguous than "compiler".

Reply:
Yeah I would agree with that. Perhaps we should consider adding `Cats` and `CatsEffect` to the other names after all

Reply:
Unless there's a way to do scalafix rule aliases and deprecations, I think it's ok and we should prioritize compatibility :) (also, we now ship this in sbt-typelevel so it would force a bump to 0.5.0 there).

As we add more rules we can be conscious about naming and think more carefully about this.